ios/react-native-navigation-sdk/NavView.m

```diff
@@ -36,8 +36,11 @@ - (void)layoutSubviews {
   }
 }
 
-- (NavViewController *)initializeViewControllerWithStylingOptions:(NSDictionary *)stylingOptions {
+- (NavViewController *)initializeViewControllerWithStylingOptions:(NSDictionary *)stylingOptions
+                                                     fragmentType:(NSInteger)fragmentType {
   _viewController = [[NavViewController alloc] init];
+  // FragmentType 0 = MAP, 1 = NAVIGATION.
+  _viewController.isNavigationEnabled = fragmentType == 1;
```

Comment:
Same as above. Let's please define constants for the values.

Reply:
Done.